package.json

```diff
@@ -29,13 +29,13 @@
   },
   "dependencies": {
     "cli-usage": "^0.1.1",
-    "clone": "^0.1.18",
+    "clone": "^1.0.2",
     "growly": "^1.2.0",
-    "lodash.clonedeep": "^3.0.0",
+    "lodash.clonedeep": "^3.0.1",
```

Comment:
Good catch @stevemao! No need for both. I'm thinking the best way to do this is to just use `lodash.clonedeep` for all. This is a dependency that might be deduped the most (I think).

Also, I don't think there is any need to bump the `lodash.clonedeep` dependency. This only means that fewer dependents of node-notifier can dedupe `lodash.clonedeep`. In general, libraries and dependencies should use dependencies that have the lowest possible version they can use and allow up for the next major release. This will allow for most possible "module reuse".